src/main/scala/dibl/proto/TilesConfig.scala

```diff
@@ -54,6 +54,9 @@ import scala.scalajs.js.annotation.{ JSExport, JSExportTopLevel }
   val shiftColsSE: Int = queryFields.getOrElse("shiftColsSE", centerMatrixCols.toString).safeToInt
   val shiftColsSW: Int = queryFields.getOrElse("shiftColsSW", "0").safeToInt
 
+  @JSExport
+  val tileDef: String = queryFields.getOrElse("tile", "-")
```

Comment:
I don't understand what you are asking.  I have moved the tileDef export to a higher line to separate it from the tile translation information.

Reply:
I'm very sorry, I meant line 25 not 55

https://github.com/d-bl/GroundForge/blob/dd6945d6fc3bf9b5b433433225240a3816488390/src/main/scala/dibl/proto/TilesConfig.scala#L25

Creating a TilesConfig class is also quite expensive to get just one argument out of the urlQuery. JavaScript can be slow under circumstances.

Reply:
Converting a scala Seq[] to a javascript array is not trivial.  Much nicer to export as a string.

Reply:
Why is config being used here in showProto if it might have changed?

  d3.selectAll("#pattern textarea").attr("rows", **config**.maxTileRows + 1)
  d3.select("#footside").attr("cols", **config**.leftMatrixCols + 2)
  d3.select("#tile"    ).attr("cols", **config**.centerMatrixCols + 2)
  d3.select("#headside").attr("cols", **config**.rightMatrixCols + 2)